### DISCUSSION TITLE
Decouple plotting from farming, node, etc

### DISCUSSION THREAD

**Discussion by joelrb** [Chia-Network/chia-blockchain#5467] [Ideas]
It is one of the biggest issues that where there's a problem syncing and force restart doesn't work we cannot restart Chia because we are plotting. It would be fine if plotting still at 5% or less but if already more than that it would be a pain to restart Chia just to re-sync.

Yesterday I restarted internet router and Chia never re-sync again. Checking the logs tells me that Chia could not recover from loss of internet connection which caused a websockets exception. The only solution is to restart Chia/GUI which is not a great option due to plotting in progress.

So, please just decouple plotting from the rest so we can restart Chia anytime there's a problem.

**Comment by Daniel-1276** [comment]
As stated im my previous [BUG](https://github.com/Chia-Network/chia-blockchain/issues/5448) report, if you use CLI for farming you will 
- be able to restart GUI client at any time,
- not see any progress bars in the GUI nor any activity at all
- you will see activity in each CLI window you start using my micro powershell script. After a couple of days you get used to it and even don't bother at all about the process.

**Reply by joelrb** [reply]
CLI is good for people like us (techie) who are comfortable coding. But for non-techie people who just want to farm from GUI too bad for them. Chia Network should give consideration to them.
